images/base/Dockerfile.ppc64le

```diff
@@ -0,0 +1,18 @@
+#
+# This is the base image from which all OpenShift Origin images inherit. Only packages
+# common to all downstream images should be here. Depends on Centos 7.2+.
+#
+# The standard name for this image is openshift/origin-base
+#
+FROM ppc64le/centos:centos7
```

Comment:
We've changed how base images work - you'd have a pp64le "source" image that is different, and everything else would be the same.